ansible/roles/config-diff-vars/filter_plugins/utils.py

```diff
@@ -44,7 +44,7 @@ def ip_mappings(context, hosts):
             continue
         result[host] = []
         for network in networks:
-            interface = net_interface(context, host, network)
+            interface = net_interface(context, host, network) or ("missinginterface" + network)
```

Comment:
```suggestion
            interface = net_interface(context, host, network)
            if not interface:
                continue
```
Would also work, but maybe that is less debug-able.